docs/UserGuide.md

```diff
@@ -85,29 +130,36 @@ Example:
 
 Precise outputs when the command succeeds:
 
-```
-Got it. I've added a buyer contact:
+>Got it. I've added a buyer contact:<br>
 Jane Doe; Phone: 91234567; Email: janedoe@gmail.com; Address: 1 College Ave East; House Info: Central Area 5 Room Condominium; Priority: high; Tags:
-```
+
 
 Precise outputs when the command fails due to missing name parameter:
-```
-Invalid command format!
-buyer: Adds a buyer to the address book. Parameters: n/NAME [p/PHONE] [e/EMAIL] [ah/ADDRESS] [i/INFO] [prio/PRIORITY] [t/TAG]... 
+
+>Invalid command format!<br>
+buyer: Adds a buyer to the address book. Parameters: n/NAME [p/PHONE] [e/EMAIL] [ah/ADDRESS] [i/INFO] [prio/PRIORITY] [t/TAG]...<br> 
 Example: buyer n/John Doe p/98765432 e/johnd@example.com ah/311, Clementi Ave 2, #02-25 i/Central Area 5 Room Condominium prio/medium t/friends t/owesMoney
-```
+
 
 Precise outputs when the command fails due to invalid parameters:
-```
-Emails must contain at least one '@'.
-```
+
+>Emails must contain at least one '@'.
+
+
+<div style='text-align: right;'>
+
+[Back to top](#rtpm-user-guide)
+
+</div>
+
+--------------------------------------------------------------------------------------------------------------------
 
 ### Adding a seller : `seller`
 
-Adds a seller with their info to the list. This command can cause warnings, and empty optional parameters will be omitted.
+Adds a seller with their information to the list. 
 
 Format: `seller n/NAME [p/PHONE_NUMBER] [e/EMAIL] [ah/HOME_ADDRESS] [as/SELLING_ADDRESS] [i/SELLING_HOUSE_INFO] [prio/PRIORITY] [t/TAG]…`
-- `n/NAME`: Contains at least one non-whitespace
+- `n/NAME`: Contains at least one letter, number, or symbol.
```

Comment:
By the way do you think it would be good to have this in an appendix at the end? that way we don't need to repeat it for every create/update command

Reply:
That does make sense, but right now it's only iterated once for each add buyer/seller command and the edit commands tell the user to refer to those; the appendix would also be rather short so lacks a bit of value. Perhaps an improvement could be to group the "no restrictions" fields together or omit them to reduce clutter?